lib/HackTest.rb

```diff
@@ -17,7 +19,13 @@ def my_handler(_event_type, _headers, data)
       data_hash[:repository][:full_name],
       data_hash[:pull_request][:number]
     ) # TODO: handle pagination
-    search_files(files)
+    non_inclusive_words_found = search_files(files)
+
+    if data_hash.key?(:pull_request) &&
+        (data_hash[:action] == 'opened' || data_hash[:action] == 'ready_for_review' || data_hash[:action] == 'reopened' || data_hash[:action] == 'edited')
```

Comment:
/nit would be nice to extract the list of actions that we care about to an Array constant and then just check if `data_hash[:action]` is in that list

Reply:
Also I don't think 'ready_for_review' is necessary here? If I'm understanding these events correctly, we'd have already received an 'opened' event when the draft PR was opened, changing it to 'ready_for_review' doesn't mean the content has changed, it's just moved out of draft status.